pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -328,6 +328,7 @@ public boolean commitSegment(String rawTableName, final String committingSegment
     oldSegMetadata.setEndOffset(nextOffset);
     oldSegMetadata.setStatus(CommonConstants.Segment.Realtime.Status.DONE);
     oldSegMetadata.setEndTime(now);
+    // TODO Set download URL? Should these segments be downloaded via vip?
```

Comment:
I see a couple of TODO's in the current PR. Are these being tracked under a ticket somewhere, so we can go back and make sure they are addressed?